dataland-frontend/tests/component/components/resources/frameworkDataSearch/p2p/P2pPanel.cy.ts

```diff
@@ -200,3 +198,24 @@ describe("Component test for P2pPanel", () => {
     ).to.deep.equal([firstYearObject, firstOtherObject, secondOtherObject]);
   });
 });
+
+/**
+ * Calls the testfunction for sorting and swaps the list entries if necessary.
```

Comment:
Isn't this the exact same function as in the LksgPanel.cy.ts? Extract it to some utils file then.